docs/poc2023q1/example1.md

```diff
@@ -0,0 +1,256 @@
+# 2023q1 PoC Example 1
```

Comment:
It appears the scenarios are driven by deployment strategies (Blue/Green in this case).  Do we believe this is scalable given the number of deployments an organizations will have for multiple workloads?  Aren't the differences between deployment strategies considered to be customizations on workloads?  Should this be driven by customization set and not deployment strategy?  Or are these analogous?

Reply:
Is it possible to have more than one instance of the ESPWS?  Should there be boxes stacked behind the ESPWS to show their could be more than one send spec, creating MBs, and receiving summarized status?  I see the ESPWS as a possible bottleneck here.  It should be possible to import those API into additional workspaces, no?  If so, how does a MBWS find an available ESPWS to communicate with in the case of failure?

Reply:
Dang, the choice of workload is confusing.  I am not trying to show a blue/green deployment strategy here, just two workloads with minimal text.  I will revise to be clearer.

The edge service provider workspace is the only one that is a singleton.  I do not know what you mean by "send spec".  The ESPW just has the edge controllers.

Dang, I forgot to show the creation of the relevant APIBindings in the workload management workspaces.  I will fix that.

The workload management workspace does not have any component that opens connections to edge controllers.  HA of the edge controllers is a separate matter.  HA within a region is easily managed with regular technology.  Multi-region HA of the edge controllers is beyond the scope of this PoC.

Reply:
Done in the latest text (still need to update pictures).

Reply:
I will wait for revisions to chime in.  Thanks

Reply:
The text revisions are done.
The pictures need to be udpated to follow the renamings: kind1->florin, kind3->guilder, blue->common, green->special.

Reply:
The pictures have been revised.